src/interface.jl

```diff
@@ -430,6 +433,21 @@ end
 @doc "$(_doc_exponential)"
 ManifoldsBase.exp!(G::AbstractLieGroup, ::Any, ::Any)
 
+function ManifoldsBase.get_coordinates(G::AbstractLieGroup, g, X, B::AbstractBasis)
```

Comment:
```suggestion
function ManifoldsBase.get_coordinates(G::AbstractLieGroup, g, X, B::AbstractBasis{<:Any,TangentSpaceType})
```
This restricts the method to based of tangent spaces (rather than arbitrary bases of vector spaces) and should fix the ambiguity because the ambiguous methods are for the cotangent space.

Reply:
Ah, I See. That detail of basis I did not have in my active memory.